cmake/ecbuild_git.cmake

```diff
@@ -224,6 +228,16 @@ macro( ecbuild_git )
 
       endif() ####################################################################################
 
+      if( _PAR_RECURSIVE )
+        ecbuild_info("git submodule --quiet update --init --recursive @ ${ABS_PAR_DIR}")
+        execute_process( COMMAND "${GIT_EXECUTABLE}" submodule --quiet update --init --recursive -q
+                        RESULT_VARIABLE nok ERROR_VARIABLE error
+                        WORKING_DIRECTORY "${ABS_PAR_DIR}")
+        if(nok)
+          ecbuild_warn("git submodule update --init --recursive in ${_PAR_DIR} failed:\n ${error}")
```

Comment:
Is there any particular reason why this should not be a critical failure?

Reply:
none in particular.  The failure is now critical. Thanks for the review @oiffrig